Makefile

```diff
@@ -0,0 +1,39 @@
+PACKAGE_NAME=osbuild
+VERSION=1
+
+sdist:
+	python3 setup.py sdist
+	find `pwd`/dist -name '*.tar.gz' -printf '%f\n' -exec mv {} . \;
+
+tarball:
+	git archive --format=tar.gz HEAD > osbuild-git.tar.gz
+	mkdir osbuild-$(VERSION); pushd osbuild-1; tar xf ../osbuild-git.tar.gz
+	tar cf osbuild-$(VERSION).tar.gz osbuild-$(VERSION)
```

Comment:
Because `%autosetup` in RPM spec file expect to extract the tarball and get a directory with `name-version`. Git-archive creates a tarball without the nested directory, so I create it by hand in here.

Reply:
Ah, I see. `git archive` has a `--prefix` option which you can use for that:

```
$ git archive --prefix=osbuild-$(VERSION)/ --format=tar.gz HEAD > osbuild-git.tar.gz
$ tar --list -f osbuild-git.tar.gz
osbuild-1/
osbuild-1/.gitignore
osbuild-1/.packit.yaml
osbuild-1/.pylintrc
osbuild-1/.travis.yml
osbuild-1/LICENSE
# ...
```

Reply:
cool, now it is simplified